test/model/content_test.dart

```diff
@@ -735,14 +735,36 @@ class ContentExample {
         '<img data-original-dimensions="6000x4000" src="/user_uploads/thumbnail/2/ce/nvoNL2LaZOciwGZ-FYagddtK/image.jpg/840x560.webp"></a></div>', [
     ImagePreviewNodeList([
       ImagePreviewNode(srcUrl: '/user_uploads/2/ce/nvoNL2LaZOciwGZ-FYagddtK/image.jpg',
-        thumbnailUrl: '/user_uploads/thumbnail/2/ce/nvoNL2LaZOciwGZ-FYagddtK/image.jpg/840x560.webp',
+        thumbnail: ImageThumbnailLocator(
+          urlPath: '/user_uploads/thumbnail/2/ce/nvoNL2LaZOciwGZ-FYagddtK/image.jpg/840x560.webp',
+          hasAnimatedVersion: false,
+        ),
```

Comment:
nit: can keep these a bit more compact:
```suggestion
        thumbnail: ImageThumbnailLocator(hasAnimatedVersion: false,
          urlPath: '/user_uploads/thumbnail/2/ce/nvoNL2LaZOciwGZ-FYagddtK/image.jpg/840x560.webp'),
```